src/main/java/org/apache/sling/feature/maven/mojos/manager/FeaturesManager.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.sling.feature.maven.mojos.manager;
+
+import org.apache.sling.feature.ArtifactId;
+import org.apache.sling.feature.Feature;
+
+import java.util.Dictionary;
+
+public interface FeaturesManager {
+
+    void init(String groupId, String artifactId, String version);
+
+    Feature getTargetFeature();
+
+    Feature getRunMode(String runMode);
+
+    void addArtifact(String runMode, ArtifactId id);
+
+    void addArtifact(String runMode, ArtifactId id, Integer startOrder);
+
+    void addConfiguration(String runMode, String pid, Dictionary<String, Object> configurationProperties);
+
+    void serialize() throws Exception;
```

Comment:
This method is never used

Reply:
Sorry, my bad - I forgot to delete the 'manager' package as they are not used anymore. Will delete them right away.

Reply:
Class removed